controllers/operands/operandHandler.go

```diff
@@ -148,6 +144,34 @@ func (h *OperandHandler) addOperands(scheme *runtime.Scheme, hc *hcov1beta1.Hype
 	}
 }
 
+type GetHandler func(log.Logger, client.Client, *runtime.Scheme, *hcov1beta1.HyperConverged) (Operand, error)
+
+func (h *OperandHandler) addOperand(scheme *runtime.Scheme, hc *hcov1beta1.HyperConverged, getHandler GetHandler) {
+	handler, err := getHandler(logger, h.client, scheme, hc)
+	if err != nil {
+		logger.Error(err, "can't create ConsoleQuickStarts objects")
+		return
+	}
+
+	var (
+		obj client.Object
+	)
+
+	if gh, ok := handler.(crGetter); ok {
+		obj, err = gh.getFullCr(hc)
+	} else {
+		err = fmt.Errorf("unknown handler with type %T", handler)
+	}
+
+	if err != nil {
+		logger.Error(err, "can't create object")
```

Comment:
would you mind adding the object type we were trying to create?

Reply:
not available from here